pkg/randchar/generator.go

```diff
@@ -244,6 +244,30 @@ func NewCharset(characters string) Charset {
 	}
 }
 
+// GetCharset returns the charset with the specified name.
+func GetCharset(charsetName string) Charset {
```

Comment:
Maybe call this `CharsetByName`, to distinguish from e.g. creating a charset from all characters in the string (as `NewCharset` does).